library/translation.inc.php

```diff
@@ -301,4 +301,27 @@ function mb_strpad($input, $length, $pad = ' ', $type = STR_PAD_RIGHT, $charset
 
 return $output;
 }
-?>
+
+
+/**
+ * Returns language direction
+ * @param int $lang_id language code
+ * @return string 'ltr' 'rtl'
+ * @author Amiel <amielel@matrix.co.il>
+ */
+function getLanguageDir($lang_id) {
+    //static $rtl_lang_codes = array('he','ar');
+
+    // validate language id
+    $lang_id = empty($lang_id) ? $lang_id : (int)$lang_id;
+ 
+    // get language code
+    $res = sqlStatement("select lang_code, lang_is_rtl from lang_languages where lang_id=?",array($lang_id));
```

Comment:
And just use sqlQuery, which is easier when just getting one row back(it converts the one row to an array):
``````
$res = sqlQuery("select lang_is_rtl from lang_languages where lang_id=?",array($lang_id));
return (1 === $res['lang_is_rtl'] ) ? 'rtl' : 'ltr'; // RTL if lang_is_rtl column is 1